pep-0570.rst

```diff
@@ -345,6 +375,10 @@ goes against the spirit of PEP 399 [#PEP399]_ to avoid duplication of effort by
 mandating that all modules added to Python's standard library **must** have a
 pure Python implementation with the same interface and semantics.
 
+-------------------------
+Consistency in Subclasses
+-------------------------
+
 Another scenario where positional-only parameters provide benefit occurs when a
 subclass overrides a method of the base class and changes the name of arguments
```

Comment:
Not part of this diff, but I think I missed this in #972 to change "arguments" to "parameters" here.